src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,97 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private User firstUser;
+    private User secondUser;
+    private User thirdUser;
+    private User fourthUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        firstUser = new User("Mike", "mike12345", 22);
+        secondUser = new User("Alice", "password", 19);
+        thirdUser = new User("Melisa", "qwerty", 20);
+        fourthUser = new User("MaX", "MaX24MaX", 24);
+    }
+
+    @Test
+    void password_Length_Is_Valid() {
+        firstUser.setPassword("ddd");
+        secondUser.setPassword("asdadado");
+        thirdUser.setPassword(null);
+        assertThrows(RuntimeException.class, () -> registrationService.register(firstUser));
+        assertEquals(secondUser, registrationService.register(secondUser));
+        assertThrows(RuntimeException.class, () -> registrationService.register(thirdUser));
+    }
+
+    @Test
+    void age_Is_Valid() {
+        firstUser.setAge(10);
+        secondUser.setAge(22);
+        thirdUser.setAge(null);
+        fourthUser.setAge(-100);

```

Comment:
Same case as with passwords: create one test method for one test case. Don't combine them all at one method. It's actual thing for all of your test cases.